packages/main/jest.config.js

```diff
@@ -1,7 +1,7 @@
 module.exports = {
   preset: "ts-jest",
   runner: "@jest-runner/electron/main",
-  setupFiles: ["./jest.setup.js"],
+  setupFilesAfterEnv: ["./jest.setup.js"],
```

Comment:
I made a change a while ago that I thought would increase test timeouts so that we don't get these intermittent failures. But apparently I did it wrong. I pushed this change to your branch to fix that.